website/conferences/utils.py

```diff
@@ -67,12 +67,13 @@ def upload_attachment(user, node, attachment):
     attachment.seek(0)
     name = (attachment.filename or settings.MISSING_FILE_NAME)
     content = attachment.read()
-    upload_url = waterbutler_api_url_for(node._id, 'osfstorage', name=name, base_url=node.osfstorage_region.waterbutler_url, cookie=user.get_or_create_cookie(), _internal=True)
+    upload_url = waterbutler_api_url_for(node._id, 'osfstorage', name=name, base_url=node.osfstorage_region.waterbutler_url, cookie=user.get_or_create_cookie().decode(), _internal=True)
```

Comment:
We do this sort of thing a lot, and I'm not certain that there are no cases where varying types may be passed (not here, but as a general statement).

It would be preferable to consistently use either `ensure_str` or `ensure_bytes`, and probably to place both of those utils somewhere in `osf/utils/`, rather than having one in `addons/box/models` and the other in `framework/encryption/__init__`

It's not urgent, but please file a maintenance ticket for it